src/client/components/pages/Courses/tableFields.tsx

```diff
@@ -165,8 +172,19 @@ export const formatMeetings = (
           ))}
         </TableCellList>
         <BorderlessButton
-          onClick={(): void => {}}
+          id={`${course.id}-${sem}-edit-meetings-button`}
+          onClick={() => coursesPageContext
+            && coursesPageContext.onMeetingEdit({
+              course,
+              term: sem,
+            })}
           variant={VARIANT.INFO}
+          forwardRef={coursesPageContext
+            && coursesPageContext.currentCourseInstance
```

Comment:
What do you think about using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) here?

Reply:
Oh thanks! I thought our linter didn't like optional chaining for some reason. I updated to use optional chaining in commit e43678f